docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "tas-item",
+  "description": "A question item for TAS",
+  "definitions": {
+
+    "tas-singleLine": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-singleLine",
+      "title": "tas-string",
+      "description": "A text question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["singleLine"]
+        },
+        "minSize": {
+          "description": "Min number of characters",
+          "type": "number",
+          "default": 1
+        },
+        "maxSize": {
+          "description": "Max number of characters",
+          "type": "number",
+          "default": 140
+        }
+      }
+    },
+
+    "tas-multiline": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-multiline",
+      "title": "tas-multiline",
+      "description": "A multiline text question item",
+      "type": "object",
+      "properties": {
```

Comment:
There's too much copy and paste here. Can we have single line and multi line share a definition here (e.g. minSize)?